lib/services/news/news_feed.php

```diff
@@ -31,7 +31,7 @@ public function getNewsFeed() {
     $remoteNewsFeed = $this->remoteNewsFeedService->getFeed();
     $localNewsFeed = $this->getLocalNewsFeed();
     $aggregatedNews = array_merge($remoteNewsFeed, $localNewsFeed);
-    usort($aggregatedNews, array(self, 'aggregateNewsSort'));
+    usort($aggregatedNews, array(__CLASS__, 'aggregateNewsSort'));
```

Comment:
weird, not sure how this didn't come about in my environments. Also realized if we're going to use `self` or `__CLASS__`, then the function needs to be static, I believe. Otherwise we'd want to use `$this`.

Want to update `aggregateNewsSort` to be a static method?